pipelines/h5c/rtp/v1/h5c_rtp_stage_1.toml

```diff
@@ -271,14 +283,24 @@ args = ["{basename}",
         "${REDCAL_OPTS:max_dims}",
         "${ANT_METRICS_OPTS:extension}",
         "${A_PRIORI_STATUS_OPTS:good_statuses}",
-        "${LIBRARIAN_OPTS:upload_to_librarian}",
-        "${LIBRARIAN_OPTS:librarian_redcal_known_good}",
        ]
 # This runs redcal on 1 out of every 10 files
 chunk_size = 1
 stride_length = 10
 time_centered = false
 
+[LIBRARIAN_REDCAL_KNOWN_GOOD]
+prereqs = "REDCAL_KNOWN_GOOD"
+prereq_chunk_size = "all"
+chunk_size = 1
+stride_length = "all"
+time_centered = false
+args = ["{basename}",
+        "${LIBRARIAN_OPTS:upload_to_librarian}",
+        "${LIBRARIAN_OPTS:librarian_ant_metrics}",
```

Comment:
This line needs to be `librarian_redcal_known_good`

Reply:
Done